### ISSUE TITLE
Unable to use kubebox with k3d cluster

### ISSUE THREAD

**Issue by lduparc** [astefanutti/kubebox#67]
We are unable to use kubebox with a k3d cluster.

we've got unable to verify certificate when we try to login to the k3d cluster.

**Comment by astefanutti** [comment]
I've tried connecting to a k3d cluster. I haven't been able to reproduce the _unable to verify certificate_ error, but I've faced the issue that Kubebox does not currently support basic auth.

I've just pushed 87cbef89bc6ae2629e62c44afbb1ad2ea5c43ffe to provide support for basic auth, and been able to connect to a local k3d cluster:

<img width="998" alt="Screenshot 2019-11-14 at 13 04 47" src="https://user-images.githubusercontent.com/366207/68856100-f9676d00-06df-11ea-9e1f-33de39b83613.png">

With:

```console
$ k3d create
...
$ cat `k3d get-kubeconfig --name='k3s-default'`
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJWakNCL3FBREFnRUNBZ0VBTUFvR0NDcUdTTTQ5QkFNQ01DTXhJVEFmQmdOVkJBTU1HR3N6Y3kxelpYSjIKWlhJdFkyRkFNVFUzTXpjeU16ZzFOVEFlRncweE9URXhNVFF3T1RNd05UVmFGdzB5T1RFeE1URXdPVE13TlRWYQpNQ014SVRBZkJnTlZCQU1NR0dzemN5MXpaWEoyWlhJdFkyRkFNVFUzTXpjeU16ZzFOVEJaTUJNR0J5cUdTTTQ5CkFnRUdDQ3FHU000OUF3RUhBMElBQk0yMlpGNkpQT1YyUXNlaDNoakJlT295aGFDUVZqZjJOa3pjeXJleTl4NFIKcWduTHpPcEdmQzJoTTBKa0tPbUcwOGcwMkZTek9ZZTJHRlVUM1dTbG83YWpJekFoTUE0R0ExVWREd0VCL3dRRQpBd0lDcERBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUFvR0NDcUdTTTQ5QkFNQ0EwY0FNRVFDSUYvTVg1WUFqM0pkCm5qUy9RRWdtbkI2Sjh3ekthTWlIcW5IdFBJTkw4WUl0QWlBNzJoUDlxbyt0b2p1QTRUeFc5RjhPcWtJOGZBV2gKdiszUGtGVEpSN0V1S0E9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
    server: https://127.0.0.1:6443
  name: default
contexts:
- context:
    cluster: default
    user: default
  name: default
- context:
    cluster: default
    namespace: default
    user: default
  name: default/default/default
current-context: default/default/default
kind: Config
preferences: {}
users:
- name: default
  user:
    password: REDACTED
    username: admin
```

This will be available in the next release.

Thanks a lot for the report and your patience!

**Comment by astefanutti** [comment]
This is available in 0.7.0 release.
